tests.py

```diff
@@ -7,17 +7,25 @@
 import safeyaml
 
 SMOKE_TESTS = {
-    """ [0] """:            [0],
```

Comment:
I quite like this style of laying out maps, it's something perltidy handles quite well. Can't see any knobs in yapf to allow it though, and I don't think it's worth fussing over.

Reply:
Speaking from firsthand experience with `gofmt`'s auto-alignment, it's good for readability but bad for diffs (which is presumably PEP8's rationale for not doing it). I personally agree with `gofmt`, but unfortunately I don't think we have an easy way to do it.